### PR TITLE
Implement proposals as template partials

### PR DESCRIPTION
**Pull request by thibaudcolas**
This implements "template partials" versions of the two proposals for new template tags. This is mostly intended to help review the output markup – although the template partials versions do work, they have big limitations compared to template tags.

### CODE REVIEW

File: wagtail_picture/templates/responsive_picture.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% load wagtailimages_tags %}
+{# width- resize rules shouldn’t be hard-coded, but there is no way for us to provide it via Django Templates’ include, so we have to pretend #}
```
Comment (chris-lawton) [inline wagtail_picture/templates/responsive_picture.html:2]: so i'm presuming a future task is for us to allow `resize_rule` to be passed via the `include` and the accepted by the `{% image %}` tag?

**Comment by chris-lawton** [comment]
Looking over these they make sense to me and the output markup is what i'd expect to see if I were going to use these partials myself. I'm wondering how these would work if you wanted to use the `fill-` rules for the images? Maybe i'm thinking too far ahead.

File: wagtail_picture/templates/responsive_picture.html (hunk 1 above)
Comment (thibaudcolas) [reply wagtail_picture/templates/responsive_picture.html:2]: Yep, which is unfortunately impossible to do with `{% include %}` – hence why I think new template tags will be the only option

**Comment by thibaudcolas** [comment]
Thank you @chris-lawton, I’m merging this now so we can move on to the next step.

Support for the different `width-` and `fill-` rules is a real flaw with this approach. Practically I think this makes `{% include %}` template partials unsuitable for what we’re after here, hence why we’ll propose a template tag. I think the markup should be the same though.

**Pull request merged**